### ISSUE TITLE
Checklists not keyboard accessible on Electron apps

### ISSUE THREAD

**Issue by msilbers** [Automattic/simplenote-electron#1265]
Adding this as a bug instead of a feature request since it deals with Accessibility.

From a user:
> In the Electron app (probably affects web too), you can't toggle a checklist item with a keyboard. This is important for screen reader users who don't use the mouse. If you could just add a way to display the actual [ ] and [x] instead of a fancy checkbox, everything would be fine. Making the checkbox keyboard focusable woudl also help. That should be the preferred way, because it's compliant with accessibility standards. A less preferred (harder to discover) solution could involve a separate shortcut to toggle the currently selected item.

### Expected
To be able to check off a checklist item with the keyboard, as not everyone uses a mouse.

### Observed
Can check off checklist items only by clicking them. Also, although you can add items with markdown, they display as checkboxes right away, so they cannot be edited without being removed.

MacBook Pro | MacOS 10.14.4 | Simplenote 1.5.0
MacBook Pro | MacOS 10.14.4 | Google Chrome Version 74.0.3729.131 | Simplenote web (new)

#2020346-zen for support reference
